defactor-com/ui-kit
# Feat/test in table component 81

## Fix requested changes

### What does this PR do?

- Resolve #81 

### Steps to test

1. Run yarn storybook
2. Check table item

Before:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/aaee3d47-68a2-4d55-afe7-af76f5e2b8e1)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/505431ed-3a4b-4593-88ae-f7584163beb6)

Now:
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/91a91b26-9147-4ff7-a1da-a2cbd0cd49de)
![imagen](https://github.com/defactor-com/ui-kit/assets/49013556/4df1809c-cb1b-4486-9ee8-cdce72c39ed0)


#### CheckList
